backend/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 spring.data.mongodb.uri=${MONGODB_URI}
 app.dalle.api.url=https://api.openai.com/v1/images
 app.openai.api.key=${APP_OPENAI_API_KEY}
-app.openai.api.org=${APP_OPENAI_API_ORG}
\ No newline at end of file
+app.openai.api.org=${APP_OPENAI_API_ORG}
+CLOUDINARY_URL=${CLOUDINARY_URL}
```

Comment:
Ich kenne mich leider mit Cloudinary nicht aus...

Was mich aber interessiert hat: Wieso definierst du die property hier so als Constant? Sucht Cloudinary sie sich so in diesem Format?

Reply:
Ja. In der URL sind Key und Secret hinterlegt.